### PR TITLE
First draft of multiple-arg functions lesson

### PR DESCRIPTION
**Pull request by mdippery**
cc: @AnnJohn @JimCampagno

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,144 @@
...
+## Functions and Arguments
+
+In the last lesson, you learned how to create your own functions to keep self-contained chunks of code in one place, and call them as often as you want. You even learned how to create a function that takes an argument, so you can generalize its behavior a bit. Hopefully you saw how useful functions can be when writing code in Swift, but now you're about to see how they can be even _more_ useful.
+
+Let's start, though, by reviewing how you create functions.
+
+Here's your goal: Write a function that will print the string "There are &lt;number&gt; moons orbiting &lt;planet&gt;." This will let you show off to our friend Neil DeGrasse Tyson how much you know about astronomy, _and_ write some Swift code in the meantime. Start by proving how much you know about Jupiter (hint: Jupiter has _67_ moons).
+
+![67 moons, ah ah ah](.images/count.jpg)
+
```
Comment (JimCampagno) [inline README.md:29]: haha! So good.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,144 @@
...
+A simple way to write a function to print "There are 67 moons orbiting Jupiter." would be this:
+
+```swift
+func moonsOfJupiter() {
+    print("There are 67 moons orbiting Jupiter.")
+}
+```
+
+Take a moment to remember how you write functions in Swift. Functions are introduced using the `func` keyword. Then, you write the name of the function—in this case, `moonsOfJupiter` (remember, function names can't contain spaces!). Next comes the argument list in parentheses. In this case, `moonsOfJupiter` doesn't take any arguments, so you just write an empty set of parentheses. Then you write the body of the function—the statements that will be executed when the function is called—between curly braces.
+
```
Comment (JimCampagno) [inline README.md:41]: Love it. I like reinforcing this here to help them solidify their knowledge on functions.

File: README.md [hunk 3]
```diff
@@ -1,42 +1,144 @@
...
+```swift
+var planet = "Mars"
+planetDescription(planet)
+```
+
+Did you see "There are 67 moons orbiting Mars." in your console?
+
+Something's wrong here! There are only two planets orbiting Mars. If you tell Neil DeGrasse Tyson that there are 67 moons orbiting Mars, he won't be impressed at all! You need to fix this, before he can see your program!
+
```
Comment (JimCampagno) [inline README.md:84]: The flow of how you introduce stuff in this reading is perfect.

**Comment by mdippery** [comment]
Excellent!

**Pull request merged**